pkg/destroy/gcp/iam.go

```diff
@@ -59,3 +61,63 @@ func (o *ClusterUninstaller) destroyServiceAccounts() error {
 	}
 	return aggregateError(errs)
 }
+
+func (o *ClusterUninstaller) getProjectIAMPolicy() (*resourcemanager.Policy, error) {
+	o.Logger.Debug("Fetching project IAM policy")
+	ctx, cancel := o.contextWithTimeout()
+	defer cancel()
+	req := &resourcemanager.GetIamPolicyRequest{}
+	policy, err := o.rmSvc.Projects.GetIamPolicy(o.ProjectID, req).Context(ctx).Do()
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to fetch project IAM policy")
+	}
+	return policy, nil
+}
+
+func (o *ClusterUninstaller) setProjectIAMPolicy(policy *resourcemanager.Policy) error {
+	o.Logger.Debug("Setting project IAM policy")
+	ctx, cancel := o.contextWithTimeout()
+	defer cancel()
+	req := &resourcemanager.SetIamPolicyRequest{Policy: policy}
+	_, err := o.rmSvc.Projects.SetIamPolicy(o.ProjectID, req).Context(ctx).Do()
+	if err != nil {
+		return errors.Wrapf(err, "failed to set project IAM policy")
+	}
+	return nil
+}
+
+func (o *ClusterUninstaller) clearIAMPolicyBindings(policy *resourcemanager.Policy) bool {
```

Comment:
this doesn't fit the criteria of "method for ClusterUninstaller".. can we convert this to a function with more explicit API.

Reply:
done